### DISCUSSION TITLE
Is it possible to add swap the clothes?

### DISCUSSION THREAD

**Discussion by cdaniel2d** [lllyasviel/Fooocus#2159] [Q&A] [answered]
For example, if i generate a model, can I use another image of a person's clothes or just the image of the clothes to put it on my generated model? Sorry for the noob question or if this was asked already.. i'm new to this!

**Comment by mashb1t** [comment, marked as answer]
Yes, this is possible using https://github.com/lllyasviel/Fooocus/pull/2032 (not yet merged), see especially https://github.com/lllyasviel/Fooocus/pull/2032#issuecomment-1910798968 and https://github.com/lllyasviel/Fooocus/pull/2032#issuecomment-1909107094.
You can either apply the changes locally, check out [my fork ](https://github.com/mashb1t/Fooocus) where the feature is already included, or use inpainting on clothes you want to change.

**Comment by cdaniel2d** [comment]
thank you @mashb1t. from what i see in your post, looks amazing. i've never done forking before, not sure what i need to do but i tried to clone the Fooocus repo and then pull from your fork, however I'm getting tons of errors.. do i need to change any parameters in the config or how would i install it properly? would appreciate some advice

for example:
G:\focus_mashb1t>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --always-download-new-model
Update failed.
reference 'refs/heads/master' not found
Error checking version for diffusers: No package metadata was found for diffusers
Installing requirements
Couldn't install requirements.

Exception in thread Thread-4 (worker):
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "G:\focus_mashb1t\Fooocus\modules\async_worker.py", line 46, in worker
    from modules.censor import censor_batch
  File "G:\focus_mashb1t\Fooocus\modules\censor.py", line 4, in <module>
    from diffusers.pipelines.stable_diffusion.safety_checker import StableDiffusionSafetyChecker
ModuleNotFoundError: No module named 'diffusers'

**Reply by mashb1t** [reply]
There seems to be a general issue with the package installation, already reported in https://github.com/mashb1t/Fooocus/issues/9. I'll have a look and let you know.
You're also using master branch, where only main exists => please change your branch to main.
